h/templates/emails/reply_notification.html.jinja2

```diff
@@ -1,23 +1,14 @@
-<p>
-  <a href="{{ reply_user_url }}">{{ reply_user.username }}</a>
-  has just left a reply to your annotation on
-  &ldquo;<a href="{{ document_url }}">{{ document_title }}</a>&rdquo;.
-</p>
-<p>
-  On <a href="{{ parent_url }}">{{ parent.created | human_timestamp }}</a>
-  {{ parent_user.username }} said:
-</p>
-<blockquote>&ldquo;{{ parent.text or "" }}&rdquo;</blockquote>
-<p>
-  On <a href="{{ reply_url }}">{{ reply.created | human_timestamp }}</a>
-  {{ reply_user.username }} said:
-</p>
-<blockquote>&ldquo;{{ reply.text or "" }}&rdquo;</blockquote>
-<p><a href="{{ reply_url }}">Reply to {{ reply_user.username }}</a>.</p>
-<hr>
-<p>
-  <small>
-    If you&rsquo;d rather not receive these notifications you can
-    <a href="{{ unsubscribe_url }}">unsubscribe now</a>.
-  </small>
-</p>
+<p>{{ reply_user.username }} has
+<a href="{{ reply_url }}">left a reply to your annotation</a>
+on &ldquo;{{ document_title }}&rdquo;:</p>
```

Comment:
Wait, I'm confused. `reply_url` is either the in context link to the reply, or the individual page of the reply. When `document_title` falls back to being a URL it'll be the URL of the annotated document (`notification.parent.target_uri`). Presumably we don't want `<a href="{{ in context link to reply }}">{{ URL of document }}</a>`. Did you mean that we should link `document_title` to the URL of the document (`notification.parent.target_uri`)?